scripts/build/nightly/configure_clang.sh

```diff
@@ -1,60 +0,0 @@
-BOOST_DIR="${HOME}/CompiledLibs/boost_1_59_0"
```

Comment:
You should not delete those files!

Reply:
Ohh, I thought them to be local files that somehow escaped my git-ignore settings. I am very sorry. I will try and revert this immediately.